service/src/main/java/ru/karalionak/rest/service/CertificateService.java

```diff
@@ -0,0 +1,22 @@
+package ru.karalionak.rest.service;
+
+import ru.karalionak.rest.entity.Certificate;
+import ru.karalionak.rest.query.CertificateColumns;
+import ru.karalionak.rest.query.Direction;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public interface CertificateService {
+
+    void create(Certificate certificate);
+
+    void update(long id, Certificate certificate);
```

Comment:
You can provide id already with certificate entity. There is no need in 2 parameters.